pylops/linearoperator.py

```diff
@@ -313,11 +322,16 @@ def _rmatmat(self, X: NDArray) -> NDArray:
         return y
 
     def __mul__(self, x: Union[float, LinearOperator]) -> LinearOperator:
-        y = super().__mul__(x)
```

Comment:
I don't understand these changes.. in the past, we needed to check if an operator was a spicy operator and wrap it into a pylops linear operator... this is what `aslinearoperator` is mostly for... 

This:
```
if isinstance(y, LinearOperator):
    y = aslinearoperator(y)
```
doesn't really make any sense to me as if you look at what `aslinearoperator` does this is the code:

```
if isinstance(Op, LinearOperator):
   return Op
else:
   return LinearOperator(Op)
```
but since you already do the same check you are guaranteed already that the return will be Op